src/constants/logURLTemplates.ts

```diff
@@ -0,0 +1,57 @@
+import { evergreenURL, logkeeperURL } from "utils/environmentVariables";
+import { stringifyQuery } from "utils/query-string";
+
+const getEvergreenTestLogURL = (
+  taskID: string,
+  execution: string,
+  testID: string
+) => {
+  const params = {
+    test_name: testID,
+    text: true,
+  };
+  return `${evergreenURL}/test_log/${taskID}/${execution}?${stringifyQuery(
+    params
+  )}`;
+};
+
+/**
+ *
+ * @param buildID - the build ID of the resmoke job
+ * @param testID - the test ID of the resmoke job
+ * @returns `/build/${buildID}/test/${testID}`
+ */
+const getResmokeLogURL = (buildID: string, testID?: string) => {
+  const params = {
+    raw: 1,
+    s3: true, // TODO: Remove this once this becomes the default
```

Comment:
nit. Maybe if we have a ticket number for this, we can add it :0

Reply:
I think it was removed between when i started this pr and now. Removed it!